docs/pipeline-standards.md

```diff
@@ -15,241 +15,244 @@ Libuit KG, Guthrie J, Ambrosio F, Kapsak C, Unal Gultekin, Holmes J, Wright S, N
+- Defining the documentation scope: Identify what aspects of the tool's core functionality need to be documented and what format the documentation should take. This can include things like user guides, reference manuals, and API documentation.
+- Establishing documentation guidelines: Develop a set of guidelines or standards for documenting the tool's core functionality. 
+- Creating a documentation template: Develop a template or set of templates that can be used to create consistent and accurate documentation..
+- Reviewing and update the documentation: Regularly review and update the documentation to ensure that it is accurate and up to date. This can be done by gathering feedback, monitoring usage data, and making adjustments as necessary.
+- Keeping it accessible: Make the documentation easily accessible to users by providing it in different formats like HTML, PDF, and user-friendly formats.
 
-### Software Testing
-_Are there automated and/or manual tests described so that the functionality of the software can be assessed?_
+Effective pipeline documentation encompasses a broad range of practices, each targeting specific aspects of usability, transparency, and collaboration. These documentation practices ensure that users and contributors have a clear understanding of the pipeline's development, usage, and governance. By incorporating these elements, documentation becomes a comprehensive resource that supports the pipeline's integrity, facilitates community contribution, and enhances user engagement, making it an indispensable part of best practices in bioinformatics pipeline development.
 
-Including software tests for public health bioinformatics software ensures the reliability and accuracy of the tools, enhancing user confidence and promoting consistent research outcomes. These tests also facilitate early detection and resolution of potential issues, contributing to the overall stability and robustness of the software in a rapidly evolving public health landscape. At a minimum, software being implemented for public health pathogen genomics should include: 
-- Smoke tests to ensure that the basic functionality of the program is working correctly
-- Unit tests to test individual code units
-- System tests/end-to-end tests to assess the overall functionality of the program, with a focus on common and important paths
-- Regression tests to ensure that changes to the code do not break existing functionality (note: system tests can be utilized to implement regression tests)
+**To adhere to this best practice and verify adherence to this best practice:** Refer to the documentation practices listed below.
```

Comment:
Maybe we should explain how this Pipeline Documentation is going to be evaluated. Is it going to be mandatory to pass `10. Pipeline Documentation` to have all 10a to 10h passed before?

Reply:
Added clarification